File: swiftSchema/NotificationCreate.swift
import Foundation

struct NotificationCreate: Codable, Identifiable, Equatable {
	var id = UUID()

	var user_id: Int?
	var house_id: Int?
	var signal_id: Int?
	var device_type: Int?
	var action: Int?
	var action_payload: String?
	var priority: Int?
	var scope: Int?
	var free_text: String?

	private enum CodingKeys: String, CodingKey {
		case user_id, house_id, signal_id, device_type, action, action_payload, priority, scope, free_text
	}
}

File: klaxonModels/NotificationCreate.kt
package io.homemetrics.homemetrics.schemas

import com.beust.klaxon.Klaxon

data class NotificationCreate(
	val user_id: Integer?,
	val house_id: Integer?,
	val signal_id: Integer?,
	val device_type: Integer?,
	val action: Integer?,
	val action_payload: String?,
	val priority: Integer?,
	val scope: Integer?,
	val free_text: String?
)